### DISCUSSION TITLE
In editor, when switching beat snaps, save the previous beat snap used

### DISCUSSION THREAD

**Discussion by Xinnoh** [ppy/osu#23463] [Ideas] [answered]
When switching between common and triplet, it always goes back to 1/1 which literally no one uses.
Eg I only use 1/4 and 1/6 in this map, switching between the two just has an extra step.

https://github.com/ppy/osu/assets/30382015/a1002f8a-b0a2-4905-9ed8-ed20a2238661

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/23722
